pkg/frontend/adminactions/kubeactions.go

```diff
@@ -35,8 +38,9 @@ type kubeActions struct {
 
 	gvrResolver dynamichelper.GVRResolver
 
-	dyn       dynamic.Interface
-	configcli configclient.Interface
+	dyn           dynamic.Interface
+	configcli     configclient.Interface
+	kubeInterface kubernetes.Interface
```

Comment:
nit: `kubeInterface` is not an interface. It is an instance that implements `kubernetes.Interface`. We usually call it `kubernetescli` or `kubecli`. I think it will be better in line with `configcli`

```suggestion
	kubernetescli kubernetes.Interface
```

Reply:
done